automation/jleechanorg_pr_automation/tests/test_actionable_counting_matrix.py

```
@@ -25,6 +24,7 @@ def setUp(self):
 
     def tearDown(self):
         """Clean up test files"""
+        import shutil
```

Ruff (0.14.8)

`import` should be at the top-level of a file

(PLC0415)